Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Store/EventChannelSlice.ts

```diff
@@ -61,6 +61,7 @@ export const dBAction = createAsyncThunk(`EventChannels/DBAction`, async (args:
     return await handle
 });
 
+export const setChanged = createAsyncThunk(`${name}/SetChanged${name}`, async (args: void, { }) => { return; });
```

Comment:
Why is this a Thunk? can't we just export a regular Action If it does not include a Promise?